RecoMuon/DetLayers/src/MuonME0DetLayerGeometryBuilder.cc

```diff
@@ -73,7 +73,7 @@ MuonME0DetLayerGeometryBuilder::buildLayer(int endcap,
     vector<const GeomDet*> frontDets, backDets;
       
     for(std::vector<int>::iterator chamber=chambers.begin(); chamber<chambers.end(); chamber++) {
-      ME0DetId me0Id(endcap,1,(*chamber), 0);
+      ME0DetId me0Id(endcap,1,(*chamber), 1);  
```

Comment:
Please clarify why this was needed.
Wouldn't this correspond to the roll/partition 1 volume, which has smaller boundaries than the full chamber. (reco occasionally uses the boundaries of detector geometry elements)
I'm not sure if this makes any difference in this code though.